.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: Test / benchmark / coverage
+
+on:
+  push:
+    branches:
+      - '**'
+      - '!master'
+  pull_request:
+    branches:
+      - '**'
+      - '!master'
+
+jobs:
+
+  test:
+    name: Run tests with Python [${{ matrix.python-version }}]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9-dev, pypy3]
```

Comment:
Can we also add this to the matrix?

```yaml
os: [ubuntu-latest, macos-latest, windows-latest]
```

Perhaps this block is going to be like this? 🤔 

```yaml
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        python-version: [3.6, 3.7, 3.8, 3.9-dev, pypy3]
        os: [ubuntu-latest, macos-latest, windows-latest]
```

Reply:
Yep. Added MaOS and Windows to test matrix in separate commit. Tests on all platforms are passing btw ;-) Total workflow time now is ~8 min.